### DISCUSSION TITLE
劇荒求推薦

### DISCUSSION THREAD

**Discussion by tshsu04224** [1101-GS4719-C/Python-Community-GS4719-C#2019] [General]
最近暑假快到了，想趁此機會找些好看的劇來看，想問大家有什麼推薦的嗎？我先來推薦一下，前陣子看了很紅的淚之女王👑 還有背著善宰跑，這兩部題材都蠻新穎的，怕會對還沒看過的人不小心暴雷，所以就不說太多，我個人是很喜歡 背著善宰跑 這部劇，是有關穿越時空的，一開始感覺蠻像想見你，但還是有蠻大差別的，有興趣可以去看，前兩天邊錫佑還有來台灣辦見面會，真的太帥😻

**Comment by cassy0201** [comment]
我自己前陣子也看了淚之女王，雖然還沒看完但已經知道後面的劇情了哈哈哈，真的好喜歡男女主之間的化學反應！

**Comment by stone971275** [comment]
我推薦大熊餐廳這部影集，每集時長不長，但卻把廚房內的分秒必爭的壓力以及團隊合作是時所面臨的問題，刻劃得十分完整，有些人物剛開始很不討喜，但在後來的劇情中，你會慢慢理解他們為何有這種舉動。整體來說，我覺得是很值得一看的一部影集。

**Comment by sunset513** [comment]
如果是動畫的話我私心推薦 《奧術》，這部的作畫品質十分精緻，人物間的情感糾纏也渲染的十分到位，你絕對能被角色們感動！

另外，更重要的是，沒意外的話今年就會推出續集！看完了前一季之後馬上就能接著最新的進度，真心推薦你一定要看～

**Comment by YeeDaniel** [comment]
這邊我先推【清潭國際高等學校】，對於劇齡可能不到半年的我來說這部真的是摯愛！女主女2女3之間的爭鬥真的超級好看！

**Comment by yubo0oo** [comment]
the 8 show 
很深刻的刻畫，我看完很有感觸
我相當推薦

**Comment by 155Julia** [comment]
非常推薦三體這部影集，結合了東方跟西方，還有現代科技及古代文明，我讓為這部影集完美融合了所有面相，既刺激又意想不到。電影方面，侏羅紀公園、侏羅紀世界，還有暮光之城，我認為這兩個系列真是經典中的經典，百看不膩，我起碼看了輛次以上！
